wire-ios/Wire-iOS/Sources/Helpers/ColorScheme.swift

```diff
@@ -20,12 +20,19 @@ import Foundation
 import UIKit
 import WireUtilities
 import WireCommonComponents
+import SwiftUI
 
 @objc
 enum ColorSchemeVariant: UInt {
     case light, dark
 }
 
+extension Color {
+    static var backgroundColor = Color(red: 0.93, green: 0.94, blue: 0.94)
```

Comment:
is it new colors or do we have them in UIKit? If the latest better to have it define once and translated with an helper